packages/schema/lib/schemas/InputFieldSchema.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+const RefResourceSchema = require('./RefResourceSchema');
+const FieldChoicesSchema = require('./FieldChoicesSchema');
+const FieldSchema = require('./FieldSchema');
+const FieldMetaSchema = require('./FieldMetaSchema');
+
+module.exports = makeSchema(
+  {
+    description: 'Field schema specialized for input fields.',
```

Comment:
```suggestion
    description: 'Field schema specialized for input fields. ${FieldSchema.schema.description}',
```

Based on this suggestion https://github.com/zapier/zapier-platform/pull/957/files#r1940592149